homework6/task2.js

```diff
@@ -0,0 +1,12 @@
+function countToZero(num) {
+    for (let a = num; a >= 0; a--) {
+        console.log(a);
+    }
+}
+countToZero(5);
+function recursiveCounter(num) {
+    if (num > 0) {
+        recursiveCounter(num - 1);
+    }
+}
```

Comment:
add console.log to print number on each call

Reply:
```
function recursiveCounter(num) {
console.log(num);
    if (num > 0) {
        recursiveCounter(num - 1);
    }
}
```